pkg/agent/main.go

```diff
@@ -111,7 +112,7 @@ func main() {
 	localClient, err := dynamic.NewForConfig(cfg)
 	exitOnError(err, "Error creating dynamic client")
 
-	logger.Info("Starting submariner-lighthouse-agent")
+	logger.Infof("Starting submariner-lighthouse-agent version: %v", version)
```

Comment:
Grammatically, this should be

```suggestion
	logger.Infof("Starting submariner-lighthouse-agent version %v", version)
```

but to keep it consistent with other version output, and perhaps simplify parsing should we want to extract this in `subctl`, let’s make it

```suggestion
	logger.Infof("submariner-lighthouse-agent version: %v", version)
```